docs/modules/ROOT/pages/subscriptions/events/connect.adoc

```diff
@@ -0,0 +1,698 @@
+}
+union Director = Person | Actor
+type Actor {
+    name: String
+}
+type Person {
+    name: String
+    reputation: Int
+}
+interface Directed @relationshipProperties {
+    year: Int!
+}
+```
+
+A subscription to any `Movie` connections would look like:
+```graphql
+subscription {
+    movieRelationshipCreated {
+        event
+        timestamp
+        movie {
+            title
+            genre
+        }
+        createdRelationship {
+           directors {
+                year
+                node {
+                    ... on PersonEventPayload { # generated type
+                        name
+                        reputation
+                    }
+                    ... on ActorEventPayload { # generated type
+                        name
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+==== Interface Example
+Considering the following type definitions:
+```graphql
+type Movie {
+    title: String
+    genre: String
+    reviewers: [Reviewer!]! @relationship(type: "REVIEWED", properties: "Review", direction: IN)
+}
+interface Reviewer {
+    reputation: Int!
+}
+type Magazine implements Reviewer {
+    title: String
+    reputation: Int!
+}
+type Influencer implements Reviewer {
+    name: String
+    reputation: Int!
+}
+interface Review {
+    score: Int!
+}
+```
+
+A subscription to any `Movie` connections would look like:
+```graphql
+subscription {
+    movieRelationshipCreated {
+        event
+        timestamp
+        movie {
+            title
+            genre
+        }
+        createdRelationship {
+            reviewers {
+                score
+                node {
+                    reputation
+                    ... on MagazineEventPayload { # generated type
+                        title
+                        reputation
+                    }
+                    ... on InfluencerEventPayload { # generated type
+                        name
+                        reputation
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+[[non-reciprocal-relationships]]
+=== Non-reciprocal relationships
+
+Considering the following type definitions:
+```graphql
+type Movie {
+    title: String
+    genre: String
+    actors: [Actor] @relationship(type: "ACTED_IN", direction: IN, properties: "ActedIn")
+    directors: [Director!]! @relationship(type: "DIRECTED", properties: "Directed", direction: IN)
+}
+type Actor {
+    name: String
+    movies: [Movie!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: OUT)
+}
+type Person {
+    name: String
+    reputation: Int
+}
+union Director = Person | Actor
+interface ActedIn @relationshipProperties {
+    screenTime: Int!
+}
+interface Directed @relationshipProperties {
+    year: Int!
+}
+```
+
+Based on the type definitions, we can make some observations:
+
+1. A `Movie` defines a relationship to `Actor` of type `ACTED_IN`
```

Comment:
Maybe we could remove some of these in-depth explanations, as I think most people will be able to understand the idea of a non-reciprocal relationship with an example